Jenkinsfile

```diff
@@ -12,20 +12,91 @@ pipeline {
     cron '@midnight'
   }
 
+  parameters {
+    string(name: 'engineListUrl',
+      description: 'Engine to use for build',
+      defaultValue: 'https://jenkins.ivyteam.io/job/ivy-core_product/job/master/lastSuccessfulBuild/')
+
+    choice(name: 'deployProfile',
+      description: 'Choose where the built plugin should be deployed to',
+      choices: ['sonatype.snapshots', 'maven.central.release'])
+
+    choice(name: 'deployArtifact',
+      description: "Choose which project should be released (onyl if you choose 'maven.central.release' as deployProfile)",
+      choices: ['web-tester', 'primeui-tester'])
+
+    string(name: 'nextDevVersion',
+      description: "Next development version used after release, e.g. '7.3.0' (no '-SNAPSHOT').\nNote: This is only used for release target; if not set next patch version will be raised by one",
+      defaultValue: '' )
+  }
+
   stages {
-    stage('build and deploy') {
+    stage('snapshot build') {
+      when {
+        expression { params.deployProfile != 'maven.central.release' }
+      }
+      steps {
+        script {
+          withCredentials([string(credentialsId: 'gpg.password.supplements', variable: 'GPG_PWD'), file(credentialsId: 'gpg.keystore.supplements', variable: 'GPG_FILE')]) {
+
+            def phase = env.BRANCH_NAME == 'master' ? 'deploy site-deploy' : 'verify'
```

Comment:
site-deploy is something specific for the project-build-plugin (mojo plugin doc)
... i think here we never must go farther than 'deploy'